### PR TITLE
FOSSA scan enabled

### PR DESCRIPTION
**Pull request by idvoretskyi**
Ref.: https://github.com/cncf/foundation/issues/109

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

**Comment by k0da** [comment]
@idvoretskyi Hi, can't [this](https://github.com/marketplace/actions/fossa-action) action be used instead?

**Comment by idvoretskyi** [comment]
@k0da we can use it, I have no objections.

The workflow in the proposed PR is the same we've used for a while across various CNCF projects, AFAIK https://github.com/marketplace/actions/fossa-action didn't exist at that time. Thanks for pointing it out.

**Comment by idvoretskyi** [comment]
@k0da updated, let's see how it goes.

**Review by k0da — DISMISSED** [review]
LGTM

File: .github/workflows/fossa.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run FOSSA scan and upload build data
+        uses: fossa-contrib/fossa-action@v1
+        with:
+          fossa-api-key: 044cfa03c61e6271a24349184e90b381
```
Comment (kuritka) [inline .github/workflows/fossa.yml:18]: @idvoretskyi , please, enter the last line.

File: .github/workflows/fossa.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: FOSSA License Scanning
+
+on:
+  - pull_request
+  - push
```
Comment (kuritka) [inline .github/workflows/fossa.yml:5]: Compatibility with build, terratest workflows.  @k0da , @ytsarev, @somaritane , thoughts? 

```suggestion
on:
  push:
    paths-ignore:
      - '**.md'
      - '**.svg'
      - '**.drawio'
      - '.spelling'
```

**Review by kuritka — CHANGES_REQUESTED** [review]
nice job @idvoretsky! please see my comments.

File: .github/workflows/fossa.yml (hunk 2 above)
Comment (ytsarev) [reply .github/workflows/fossa.yml:5]: @kuritka good point

**Comment by idvoretskyi** [comment]
@kuritka comments addressed, but I think @ytsarev has to sign-off his commit :)

**Review by kuritka — APPROVED** [review]
👍

File: README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/absaoss/k8gb)](https://hub.docker.com/r/absaoss/k8gb)
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/k8gb)](https://artifacthub.io/packages/search?repo=k8gb)
 [![doc.crds.dev](https://img.shields.io/badge/doc-crds-purple)](https://doc.crds.dev/github.com/absaoss/k8gb)
+[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb.svg?type=shield)](https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb?ref=badge_shield)
```
Comment (ytsarev) [inline README.md:16]: @idvoretskyi Is https://app.fossa.com/projects/custom%2B162%2Fgithub.com%2FAbsaOSS%2Fk8gb?ref=badge_shield supposed to be publicly available?
Currently getting
image

**Comment by ytsarev** [comment]
> @kuritka comments addressed, but I think @ytsarev has to sign-off his commit :)
> 
> UPD. Nevermind, I've squashed the commits.

cool, thanks a lot :)

File: README.md (hunk 3 above)
Comment (k0da) [reply README.md:16]: @ytsarev had the same, but also received invite to fossa, after that I'm able to see the page

Comment (idvoretskyi) [reply README.md:16]: @ytsarev it's public now, however, I've invited you all to FOSSA as well (check your inbox for an invite, or DM me via the CNCF Slack).

Comment (ytsarev) [reply README.md:16]: 👍

**Review by ytsarev — APPROVED** [review]

**Pull request merged**